index.html

```diff
@@ -3987,6 +3987,15 @@ <h3>JSON-LD</h3>
 base media type, `application/vc+ld+json`.
         </p>
 
+        <p>
+As elaborated upon in Section <a href="#static-processing"></a>, some software
+applications might not perform dynamic JSON-LD processing.
+Authors of <a>conforming documents</a> are advised that using JSON-LD keywords
+in the `@context` value to globally affect values in documents, such as
+globally setting the keywords `@base` or `@language`, might reduce
+interoperability.
```

Comment:
This is an attempt to more clearly open the door to use of `@language` or `@base` where they may increase usability for specific deployments, when used with properties that are not mapped by the default `@context` and/or with URIs that are not relative to inferred `@base`.... I am more than open to improvements on this language. I will not block if this is not merged, though I would like to have an issue (open or closed) on the topic labeled for "VCDM.next" or similar.
```suggestion
Authors of <a>conforming documents</a> are advised that interoperability
might be reduced if JSON-LD keywords in the `@context` value are used to
globally affect values in `@context` or other documents, such as by
globally setting the keywords `@base` or `@language`. Using these
keywords with specific focus, as when aligned with one or more values
of one or more other keywords, may broaden the utility of specific
`@context` files and/or <a>conforming documents</a>.
```